client-v2/src/shared/components/input/ButtonDefault.ts

```diff
@@ -54,7 +54,7 @@ const colorMap = {
   },
   deselected: {
     default: theme.colors.white,
-    primary: theme.colors.white,
+    primary: theme.colors.blackDark,
```

Comment:
This is nothing to do with this PR, but... I'm not really a fan of theme names referencing the colour they represent. That is, prefer `primary` to `red`, for example. This allows the theme to be changed more easily as we just change the value of `primary` without the fear of the name not representing the value.

Reply:
I sort of agree but also when you get down to having lots of different shades of white the names become harder and harder to represent. Maybe we could talk about that with respect to our current colour constants and whether there's a better way? It's definitely been discussed a bit already.